lib/wda/xcodebuild.js

```diff
@@ -232,6 +232,7 @@ class XcodeBuild {
     if (process.env.DEVELOPER_DIR) {
```

Comment:
I think it makes sense to clone the whole env of the parent process:

```js
const env  = Object.assign({}, process.env, {USE_PORT: this.wdaRemotePort});
```

WDYT @imurchie ?

Reply:
Yeah, that makes sense.

Reply:
i agree. anytime we send in `env` to a subproc, we should always clone `process.ENV` first. rarely do we actually mean to have a completely empty env.

Reply:
Alright, I've updated the code